rmf_human_detector_oakd/src/HumanDetector.cpp

```diff
@@ -28,6 +28,7 @@
 namespace rmf_human_detector_oakd {
 
 //==============================================================================
+// constructor called
```

Comment:
I don't think `constructor called` is the right comment here. It's merely the definition of the constructor.
```suggestion
// Constructor
```